components/admin/groups/Groups.js

```diff
@@ -20,14 +21,14 @@ function Groups({ data, isError, isLoading }) {
           </thead>
           {!isError && !isLoading ? (
             <tbody>
-              {groups.map((item) => (
+              {activeGroups.map((item) => (
                 <tr key={item.groupId.groupCode}>
                   <th scope="row">{item.groupId.groupCode}</th>
                   <td>{item.title.value}</td>
                   <td>{item.status}</td>
                   <td>
                     <a href={`/admin/groups/${item.groupId.groupCode}`}>
-                      <FontAwesomeIcon icon={faEdit} color="darkslategrey" />
+                      {/*<FontAwesomeIcon icon={faEdit} color="darkslategrey" />*/}
```

Comment:
get rid of comments

Reply:
I have left some commented code intentionally to indicate buttons that I have hidden and need to fixed later on. I'm open to removing that but we will have to take note of exactly what was removed and from where in the codebase.

Reply:
yea just add it as a TODO comment